internal/lib/checkpoint.go

```diff
@@ -217,10 +217,10 @@ func (c *ContainerServer) prepareCheckpointExport(ctr *oci.Container) error {
 	return nil
 }
 
-func (c *ContainerServer) exportCheckpoint(ctr *oci.Container, specgen *rspec.Spec, export string) error {
+func (c *ContainerServer) exportCheckpoint(ctx context.Context, ctr *oci.Container, specgen *rspec.Spec, export string) error {
 	id := ctr.ID()
 	dest := ctr.Dir()
-	logrus.Debugf("Exporting checkpoint image of container %q to %q", id, dest)
+	logrus.WithContext(ctx).Debugf("Exporting checkpoint image of container %q to %q", id, dest)
```

Comment:
If we have a context, do we then use our own log wrapper here?
```suggestion
	log.Debugf(ctx, "Exporting checkpoint image of container %q to %q", id, dest)
```

Reply:
I'd rather do that in the follow-up PRs, along with many smaller changes

Reply:
Sounds good!